Makefile

```diff
@@ -11,26 +11,14 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-all: agent runtime snapshotter
+SUBDIRS:=agent runtime snapshotter
 
-.PHONY: agent
-agent:
-	$(MAKE) -C agent
+all: $(SUBDIRS)
 
-.PHONY: runtime
-runtime:
-	$(MAKE) -C runtime
+$(SUBDIRS):
+	$(MAKE) -C $@
 
-.PHONY: snapshotter
-snapshotter:
-	$(MAKE) -C snapshotter
-
-.PHONY: proto
-proto:
-	proto/generate.sh
```

Comment:
are we throwing away proto stuff? I was thinking about moving it to `Makefile` from `generate.sh`

Reply:
+1 on this. Would rather have this in the make file rather than a `generate.sh` file.

Reply:
woops, that was a mistake! proto should still be there